### PR TITLE
MSPSDS 690 and 691: Small bugs fixed

### PR DESCRIPTION
**Pull request by aidayounesi**

### CODE REVIEW

File: web/app/views/investigations/meetings/context.html.slim [hunk 1]
```diff
@@ -15,7 +15,7 @@ main.govuk-main-wrapper role="main"
             | Who was the meeting with?
           .govuk-form-group
             label.govuk-label class="govuk-!-width-two-thirds"
```
Comment (tom-bell-softwire) [inline web/app/views/investigations/meetings/context.html.slim]: Sorry I didn't spot this before, but I think all the `label` elements in these views should be rendered via the form helper method, e.g.
```
= form.label :correspondent_name, "Name(s)", class: "govuk-label"
```

This has the benefit of correctly associating labels with the relevant input field, which has important implications for accessibility (screen readers, etc.).

Would you mind updating the labels in the content and context views covered by this review?

File: web/app/views/investigations/meetings/content.html.slim [hunk 2]
```diff
@@ -35,7 +35,7 @@ main.govuk-main-wrapper role="main"
           .govuk-form-group
             label.govuk-label class="govuk-!-width-two-thirds"
               | Notes
-            = form.text_area :details, class: "govuk-textarea", rows: 5
+            = form.text_area :details, class: "govuk-textarea", rows: 5, maxlength: 1000
```
Comment (tom-bell-softwire) [inline web/app/views/investigations/meetings/content.html.slim]: Is there an associated `validates_length_of` validation for this attribute on the `Correspondence` model?

File: web/app/views/investigations/meetings/context.html.slim (hunk 1 above)
Comment (aidayounesi) [reply web/app/views/investigations/meetings/context.html.slim]: Thank you. Done

File: web/app/views/investigations/meetings/content.html.slim (hunk 2 above)
Comment (aidayounesi) [reply web/app/views/investigations/meetings/content.html.slim]: No, it's probably a good idea to add it so the user can see the relevant error

Comment (aidayounesi) [reply web/app/views/investigations/meetings/content.html.slim]: Yes, there is

**Pull request merged**